packages/features/CalendarEventBuilder.ts

```diff
@@ -275,7 +276,10 @@ export class CalendarEventBuilder {
       !this.event.bookerUrl ||
       !this.event.title
     ) {
-      throw new Error("Missing required fields for calendar event");
+      throw new HttpError({
```

Comment:
I don't think the builder should throw an error at all, maybe return null (so consumers can handle)

Reply:
ah that's true, your feedback's now addressed!